CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -0,0 +1,110 @@
+# Changelog 1.7
+
+## 1.7.0 - 2022-06-15
+
+### Added
+
+- Add TrafficControl feature to control the transmission of Pod traffic; It allows users to mirror or redirect traffic originating from specific Pods or destined for specific Pods to a local network device or a remote destination via a tunnel of various types. ([#3644](https://github.com/antrea-io/antrea/pull/3644) [#3580](https://github.com/antrea-io/antrea/pull/3580) [#3487](https://github.com/antrea-io/antrea/pull/3487), [@tnqn] [@hongliangl] [@wenqiq])
+  * Refer to [this documentation](https://github.com/antrea-io/antrea/blob/release-1.7/docs/traffic-control.md) for more information about this feature.
+  * Refer to [this cookbook](https://github.com/antrea-io/antrea/blob/release-1.7/docs/cookbooks/ids/README.md) for more information about using this feature to provide network-based intrusion detection service to your Pods.
+- Add support for the IPsec Certificate-based Authentication. ([#3778](https://github.com/antrea-io/antrea/pull/3778), [@xliuxu])
+  * Add an Antrea Agent configuration option `ipsec.authenticationMode` to specify authentication mode. Supported options are "psk" (default) and "cert".
+  * Add an Antrea Controller configuration option `ipsecCSRSigner.autoApprove` to specify the auto-approve policy of Antrea CSR signer for IPsec certificates management. By default, Antrea will auto-approve the CertificateSingingRequest (CSR) if it is verified.
+  * Add an Antrea Controller configuration option `ipsecCSRSigner.selfSignedCA` to specify whether to use auto-generated self-signed CA certificate. By default, Antrea will auto-generate self-signed CA certificate.
+- Add the following capabilities to the AntreaPolicy feature:
+  * Add support for ICMP protocol in Antrea-native policies. ([#3472](https://github.com/antrea-io/antrea/pull/3472), [@GraysonWu])
+  * Add support for IGMP protocol in Antrea-native policies. ([#3660](https://github.com/antrea-io/antrea/pull/3660), [@liu4480])
+  * Add support for matching multicast traffic in Antrea-native policies. ([#3660](https://github.com/antrea-io/antrea/pull/3660), [@liu4480])
+  * Add support for rule-level statistics for IGMP traffic and multicast traffic. ([#3449](https://github.com/antrea-io/antrea/pull/3449), [@ceclinux])
```

Comment:
multicast and IGMP traffic?

Reply:
not sure if you use different words in the this comment and above comment intentionally, I updated all items about AntreaPolicy, PTAL